doc/user-guide/io.rst

```diff
@@ -776,6 +776,75 @@ dimensions included in ``region``. Other variables (typically coordinates)
 need to be explicitly dropped and/or written in a separate calls to ``to_zarr``
 with ``mode='a'``.
 
+.. _io.zarr.writing_chunks:
+
+Specifying chunks in a zarr store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Chunk sizes may be specified in one of three ways when writing to a zarr store:
+
+1. Manual chunk sizing through the use of the ``encoding`` argument in :py:meth:`Dataset.to_zarr`:
+2. Automatic chunking based on chunks in dask arrays
+3. Default chunk behavior determined by the zarr library
+
+The resulting chunks will be determined based on the order of the above list; dask
+chunks will be overridden by manually-specified chunks in the encoding argument,
+and the presence of either dask chunks or chunks in the ``encoding`` attribute will
+supercede the default chunking heuristics in zarr.
+
+Importantly, this logic applies to every array in the zarr store individually,
+including coordinate arrays. Therefore, if a dataset contains one or more dask
+arrays, it may still be desirable to specify a chunk size for the coordinate arrays
+(for example, with a chunk size of `-1` to include the full coordinate).
+
+To specify chunks manually using the ``encoding`` argument, provide a nested
+dictionary with the structure ``{'variable_or_coord_name': {'chunks': chunks_tuple}}``.
+
+.. note::
+
+    The positional ordering of the chunks in the encoding argument must match the
+    positional ordering of the dimensions in each array. Watch out for arrays with
+    differently-ordered dimensions within a single Dataset.
+
+For example, let's say we're working with a dataset with dimensions
+``('time', 'x', 'y')``, a variable ``Tair`` which is chunked in ``x`` and ``y``,
+and two multi-dimensional coordinates ``xc`` and ``yc``:
+
+.. ipython:: python
+
+    ds = xr.tutorial.open_dataset("rasm")
```

Comment:
Can we make this an idealized example to avoid having to download the dataset?

Reply:
I might be wrong, but don't we use this elsewhere, too? In that case, `pooch` shouldn't re-download this.